sigma/modifiers.py

```diff
@@ -235,10 +235,14 @@ class SigmaWindowsDashModifier(SigmaValueModifier):
     into /param-name while the second dash is left untouched.
     """
 
+    en_dash = chr(int("2013", 16))
```

Comment:
The default font of VSCode and basically any monospace font will display all dashes identically.
The yield line will then look like `yield from ("-", "/", "-", "-", "-")`
To avoid, someone in the future will remove the 'unnecessary
duplicates', I wanted to make these chars explicit.
I do not like it, but think it's a bit less ugly than the identically looking dashes😮‍💨

Reply:
Totally fine with this! The plain characters would be very confusing.